client/src/Frontend/Views/SendResources.tsx

```diff
@@ -153,6 +171,37 @@ function AbandonButton({
   );
 }
 
+function ExtractButton({
+  planet,
+  extracting,
+  disabled,
+}: {
+  planet?: Planet;
+  extracting: boolean;
+  disabled?: boolean;
+}) {
+  const uiManager = useUIManager();
+
+  if (!planet) return null;
+
+  const silver = planet.silver;
+
+  return (
+    <MaybeShortcutButton
+      size='stretch'
+      active={extracting}
+      onClick={() => uiManager.withdrawSilver(planet.locationId)}
+      shortcutKey={TOGGLE_WITHDRAW}
+      shortcutText={TOGGLE_WITHDRAW}
+      disabled={planet.isHomePlanet || disabled}
+    >
+      {/* <TooltipTrigger name={TooltipName.Abandon}> */}
+      {extracting ? 'Extracting' : `Extract all (${Math.floor(silver)}) silver`}
```

Comment:
```suggestion
      {extracting ? 'Extracting' : `Extract (${Math.floor(silver)}) silver`}
```
I think we should change this so `Extract All` can refer to extracting silver from multiple planets.